ingress/nginx.tmpl

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
+daemon off;
+
+error_log  /dev/stdout warn;
+pid        {{ .WorkingDir }}/nginx.pid;
+
```

Comment:
Need to increase `worker_rlimit_nofile` or we'll run out of file descriptors fast under load. I think default ulimit is 1024.

See: http://stackoverflow.com/questions/7325211/tuning-nginx-worker-process-to-obtain-100k-hits-per-min